.atomist/editors/NewSpringBootRestService.ts

```diff
@@ -131,33 +140,37 @@ class NewSpringBootRestService implements PopulateProject {
         });
     }
 
-    packageMove(project, new_package) {
-        let old_package = "com.atomist.springrest"
+    packageMove(project: Project) {
+        let oldPackage = "com.atomist.springrest"
 
         project.editWith("atomist-rugs.common-editors.PackageMove", {
-            "old_package": old_package, 
-            "new_package": new_package
+            "old_package": oldPackage, 
+            "new_package": this.root_package
         });
     }
 
-    renameServiceClass(project, new_class_name) {
-        let old_class_name = "SpringRest"
+    renameServiceClass(project: Project) {
+        let oldClassName = "SpringRest"
 
         project.editWith("atomist-rugs.common-editors.ClassRenamer", {
-            "old_class": old_class_name, 
-            "new_class": new_class_name
+            "old_class": oldClassName, 
+            "new_class": this.service_class_name
         });
     }
 
-    addCreationDateToReadme(project) {
-        let eng: PathExpressionEngine = project.context().pathExpressionEngine();
+    switchREADME(project: Project) {
+        let readmeFileName = "README.md"
+        project.deleteFile(readmeFileName)
+        let defaultREADMEPath = ".atomist/files/readme.txt"
+        project.copyEditorBackingFileOrFail(defaultREADMEPath, readmeFileName);
+    }
 
-        let readmePathExpression = new PathExpression<Project, File>(
-            "/*[@name='README.md']");
+    addCreationDateToReadme(project: Project, readme: File) {
+        readme.replace("{{project_name}}", this.project_name);
 
-        let readmeFile: File = eng.scalar(project, readmePathExpression);
+        readme.replace("{{description}}", this.description);
 
-        readmeFile.replace("{{creation_date}}", new Date().toISOString().split('T')[0]);
+        readme.replace("{{creation_date}}", new Date().toISOString().split('T')[0]);
```

Comment:
If you're going to do it this way, you might as well keep using the template.

Here's an example that uses the README from the generator's archive, replacing and excising as appropriate.